### PR TITLE
[stable/traefik]update rbac apiversion to v1.

### PR DESCRIPTION
**Pull request by zouyee**

### CODE REVIEW

**Comment by krancour** [comment]
This PR doesn't make any sense.

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.12.0
+version: 2.12.1
```
Comment (krancour) [inline stable/traefik/Chart.yaml:3]: Why are major and patch numbers both being incremented?

File: stable/traefik/templates/rbac.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
   name: {{ template "fullname" . }}
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k9s.io/v1
```
Comment (krancour) [inline stable/traefik/templates/rbac.yaml:8]: k9s? That's not a thing.

**Review by mattfarina — CHANGES_REQUESTED** [review]
This PR can't be merged until after Kubernetes 1.9 is out. rbac/v1 is supported in Kubernetes 1.8 but not 1.7. The [charts repo needs to support the latest and the previous minor releases](https://github.com/kubernetes/charts/blob/master/README.md#supported-kubernetes-versions). [Beta APIs are supported for 2 versions/6 months after the v1 is out](https://kubernetes.io/docs/reference/deprecation-policy/) so we can continue to use beta1 until after 1.9 is out. [1.9 is slated to be released on Dec 13th](https://github.com/kubernetes/features/blob/master/release-1.9/release-1.9.md). At that time if you would like to contribute rbac/v1 support it would be appreciated.

Note, at that point the chart version will likely need to be changed from the version in the PR to be newer than the chart at that time.

**Comment by viglesiasce** [comment]
This PR cannot be accepted as is due to the typos and version bump issues.

**Comment by viglesiasce** [comment]
Closing as stale.

**Pull request closed without merging**